### ISSUE TITLE
no-unsafe-any raises errors on import statements that are not typed as "any"

### ISSUE THREAD

**Issue by lddubeau** [palantir/tslint#2494]
### Bug Report

- __TSLint version__: 5.0.0
- __TypeScript version__: 2.2.2
- __Running TSLint via__: CLI: `tslint test.ts --type-check --project tsconfig.json`

#### TypeScript code being linted

```ts
import * as ts from "typescript";

// Prevent TypeScript from complaining about ts being unused.
console.log(ts.getCommentRange);
```

with `tslint.json` configuration:

```json
{
    "rules": {
        "no-unsafe-any": true
    }
}
```

I've tried with the following two `tsconfig.json`: 

```json
{
  "compilerOptions": {}
}
```

and:

```json
{
  "compilerOptions": {
    "rootDir": ".",
    "noEmitOnError": true,
    "noImplicitAny": true,
    "noImplicitReturns": true,
    "noImplicitThis": true,
    "skipDefaultLibCheck": true,
    "noUnusedParameters": true,
    "noUnusedLocals": true,
    "skipLibCheck": true,
    "strictNullChecks": true,
    "target": "es5"
  }
}
```

#### Actual behavior

The error:

```
test.ts[1, 21]: Unsafe use of expression of type 'any'.
```

#### Expected behavior

No error. I don't see why the imported symbols are somehow considered to be of type `any`. They obviously aren't since the IDE is able to perform completion that uses TypeScript type information.

**Comment by amatiasq** [comment]
Same issue here: it warns for every expression at the first line of each method

```
ERROR: src/pathfinding/pathfinding.spec.ts[16, 12]: Unsafe use of expression of type 'any'.
ERROR: src/pathfinding/pathfinding.spec.ts[16, 12]: Unsafe use of expression of type 'any'.
ERROR: src/pathfinding/pathfinding.spec.ts[16, 12]: Unsafe use of expression of type 'any'.
ERROR: src/pathfinding/pathfinding.spec.ts[16, 35]: Unsafe use of expression of type 'any'.
ERROR: src/pathfinding/pathfinding.spec.ts[16, 12]: Unsafe use of expression of type 'any'.
ERROR: src/pathfinding/pathfinding.spec.ts[16, 12]: Unsafe use of expression of type 'any'.
ERROR: src/pathfinding/pathfinding.spec.ts[20, 22]: Unsafe use of expression of type 'any'.
ERROR: src/pathfinding/pathfinding.spec.ts[20, 22]: Unsafe use of expression of type 'any'.
ERROR: src/pathfinding/pathfinding.spec.ts[20, 22]: Unsafe use of expression of type 'any'.
ERROR: src/pathfinding/pathfinding.spec.ts[20, 45]: Unsafe use of expression of type 'any'.
ERROR: src/pathfinding/pathfinding.spec.ts[20, 22]: Unsafe use of expression of type 'any'.
ERROR: src/pathfinding/pathfinding.spec.ts[20, 22]: Unsafe use of expression of type 'any'.
[...]
```

This is the code it's complaining about:

<img width="710" alt="screenshot" src="https://cloud.githubusercontent.com/assets/1533589/26307589/ac36ca08-3ef5-11e7-9638-464cfa7b9f0b.png">

**Comment by adidahiya** [comment]
this should be fixed in tslint >=5.1.0, can you try that out?
